### ISSUE TITLE
7zip fails to install via Install-Package

### ISSUE THREAD

**Issue by ejb1123** [chocolatey/chocolatey-coreteampackages#795]
When installing the 7zip package using the powershell PackageManagment module

## Expected Behavior
The zip package should be installed without any errors

## Current Behavior
When the command `Install-Package -name 7zip -Source chocolatey` is run it fails with the below error message
```
install-package : The system cannot find the file specified
At line:1 char:68
+ ... e = 'Continue'; install-package -Force -Source chcocolatey -Name python2,git,7zip,vswhere
+                     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (False C:\Chocol...ls\7zip_x64.exe
   :String) [Install-Package], Exception
    + FullyQualifiedErrorId : Process Execution Failed,Microsoft.PowerShell.Pa
   ckageManagement.Cmdlets.InstallPackage
```

## Steps to Reproduce (for bugs)

1. Install the chocolatey package provider `Install-PackageProvider -name chocolatey -Force`
2. Try to install 7zip `Install-Package -Name 7zip -Source chocolatey`

## Context
I am using this to help provision a windows serever core container

```
FROM microsoft/windowsservercore
SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'Continue';"]

RUN install-packageprovider -name chocolatey -Force; \
	Set-PackageSource -Trusted -Name "chocolatey"
RUN install-package -Force -Name "7zip"
```

## Your Environment
* Package Version used:  2.8.5.130
* Operating System and version: Windows Server 2016
* Chocolatey version:   2.8.5.130

**Comment by AdmiringWorm** [comment]
The `PackageManagment module` is not supported, please try again directly with the choco client.

> Chocolatey version: 2.8.5.130

That would not be the chocolatey version, the latest available is `0.10.7`.

> Package Version used: 2.8.5.130

That's not the latest version of 7zip (nor a valid version of that package at all).

**Comment by majkinetor** [comment]
Will close this because it is not related to `choco install`. Feel free to reopen if there is more to discuss.
